### PR TITLE
Development

### PR DESCRIPTION
**Pull request by jsariasgeek**
The project is finished. Please check and letme know

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,90 @@
...
     <title>Project: HTML Forms</title>
 </head>
 <body>
-    
+    <div class="sign-in">
+        <div class="sign-text">
+            <p>Already have an Intuit account?</p>
+            <a href="">I forgot my User ID or Password</a>
+        </div>
+        <button><img src="images/lock.png"> <p>Sign in</p></button>
```
Comment (JamesOkunlade) [inline index.html]: 1. An img element must have an alt attribute.
2. Element p cannot be a child element of a button. Remove the p tags

File: index.html [hunk 2]
```diff
@@ -4,9 +4,90 @@
...
+                    <p>quickbooks</p>
+            </div>
+            <div class="diferent-apps">
+                <img class="turbo" src="images/turbo.png" alt="">
+                    <p>TurboTax</p>
+            </div>    
+        </div>
+    </div>
+    <div class="form">
+        <form action="">
```
Comment (JamesOkunlade) [inline index.html]: The action attribute for form should not be left empty. Remove the action if there's no controller to handle the form.

File: index.html [hunk 3]
```diff
@@ -4,9 +4,90 @@
...
+                <div class="actions">
+                    <label for="password">Password</label>
+                    <input class="password" type="password" id="password" placeholder="Password">
+                </div>
+                <div class="actions">
+                    <label for="confirm-password">Confirm Password</label>
+                    <input type="password" id="confirm-password" placeholder="Password">
+                </div>
+                
+                <button class="create-account"><img src="images/lock.png"> <p>Create Account</p></button>
```
Comment (JamesOkunlade) [inline index.html]: Img element should always contain the alt attribute. 
Element p should not be a child element of a button element.

File: index.html [hunk 4]
```diff
@@ -4,9 +4,90 @@
...
+            
+        </form>
+        <footer>
+            <p>Invisible reCAPTCHA by Google <a href="">Privacy Policy</a> and <a href="">Terms of Use.</a></p>
+        </footer>
+    </div>
+
 </body>
+
+<footer  class="body-footer">
```
Comment (JamesOkunlade) [inline index.html]: The footer section should be part of the page body. Move the footer into the body of your HTML.

File: css/main.css [hunk 5]
```diff
@@ -0,0 +1,258 @@
+*{
+    padding: 0;
+    margin: 0;
+    font-family: sans-serif;
+    font-weight: 100;
+    box-sizing: border-box;
```
Comment (JamesOkunlade) [inline css/main.css:6]: Research on and specify vendor prefixes when using box-sizing, to make it compatible across all browsers.

File: css/main.css [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+button img{
+    height: 20px;
+    background: white;
+    border: 0px;
+    border-radius: 100%;
+    
+    
+}
+
+button p{
```
Comment (JamesOkunlade) [inline css/main.css]: This style would not be needed. Element p should not be a child element of a button. Make the change in your HTML and remove these styles.

File: css/main.css [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    text-align: center;
+    height: 20%;
+    font-size: 13px
+}
+.form-actions{
+
```
Comment (JamesOkunlade) [inline css/main.css]: Remove this whitespace.

File: css/main.css [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+    color: #8d9096;
+}
+
+.form footer{
+    position:relative;
+    padding: 10% 0%;
+    text-align: center;
+    font-size: 11px;
+    display: flex;
+
```
Comment (JamesOkunlade) [inline css/main.css]: Remove all unneeded whitespaces in your code.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
As we discussed on slack, these are changes you need:
your apps class shouldn't have any width in this case so please remove it
input type of form for phone should be tel
no code should be left after the closing body tag
button tag is inline-block by default so no tags that are block by default can be within -- in your case you have p tag within the button (edited) .
Please make those changes, and submit new review request.
Happy coding.

File: index.html (hunk 1 above)
Comment (jsariasgeek) [reply index.html]: Solved!

File: index.html (hunk 3 above)
Comment (jsariasgeek) [reply index.html]: Solved

File: index.html (hunk 4 above)
Comment (jsariasgeek) [reply index.html]: Solved!

File: css/main.css (hunk 5 above)
Comment (jsariasgeek) [reply css/main.css:6]: Solved!

File: css/main.css (hunk 6 above)
Comment (jsariasgeek) [reply css/main.css]: Solved!

File: css/main.css (hunk 7 above)
Comment (jsariasgeek) [reply css/main.css]: Solved!

File: css/main.css (hunk 8 above)
Comment (jsariasgeek) [reply css/main.css]: Solved!

**Review by zlayabekrija — APPROVED** [review]
Great work, thumbs up form me 👍 .
Good luck in the future.
Happy coding.

**Pull request merged**